ktor-features/ktor-auth-jwt/jvm/src/io/ktor/auth/jwt/JWTPrincipal.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.auth.jwt
+
+import com.auth0.jwt.exceptions.*
+import com.auth0.jwt.interfaces.*
+import io.ktor.auth.*
+import java.util.*
+import kotlin.reflect.*
+
+interface JWToken: Credential, Principal {
+    /**
+     * Get the value of the "iss" claim, or null if it's not available.
```

Comment:
Instead of referring to some field or something called claim, it's better to explain the meaning to the reader so one shouldn't look for the field "iss" somewhere in sourcecodes or google for it. We better to not refer to the particular implementation library.

Reply:
`iss` and claim are part of the JWT specification, you want to explain that specification in kdoc?